test/remote/gateways/remote_trexle_test.rb

```diff
@@ -0,0 +1,170 @@
+require 'test_helper'
+
+class RemoteTrexleTest < Test::Unit::TestCase
+  def setup
+    @gateway = TrexleGateway.new(fixtures(:trexle))
+
+    @amount = 100
+    @credit_card = credit_card('5555555555554444', :year => Time.now.year + 2)
```

Comment:
Use new Hash syntax here throughout this file too. (Sorry for nagging so much! 😬)